e2e/test/scenarios/onboarding/command-palette.cy.spec.js

```diff
@@ -43,16 +43,16 @@ describe("command palette", () => {
       cy.log("Should search entities and docs");
       commandPaletteSearch().type("Orders, Count");
 
-      cy.findByRole("option", { name: /Orders, Count/ }).should("exist");
+      cy.findByRole("option", { name: "Orders, Count" }).should("exist");
       cy.findByText('Search documentation for "Orders, Count"').should("exist");
 
       // Since the command palette list is virtualized, we will search for a few
       // to ensure they're reachable
       commandPaletteSearch().clear().type("People");
-      cy.findByRole("option", { name: /People/ }).should("exist");
+      cy.findByRole("option", { name: "People" }).should("exist");
```

Comment:
This was the problem

![image](https://github.com/metabase/metabase/assets/1937582/98fec8b4-bc74-4bd7-9852-d655a7733f16)

It matches multiple elements, but please be aware that the second element from the search results appear a little after the first and the last element, so when we query exactly by the name `People` we'll only get the first option briefly before the search results pop up.